examples/fan_control.py

```diff
@@ -0,0 +1,57 @@
+import colorsys
+import random
+
+from openrazer.client import DeviceManager
+from openrazer.client import constants as razer_constants
+
+# Create a DeviceManager. This is used to get specific devices
+device_manager = DeviceManager()
+
+
+print("Found {} Razer devices".format(len(device_manager.devices)))
+print()
+
+# Disable daemon effect syncing.
+# Without this, the daemon will try to set the lighting effect to every device.
+device_manager.sync_effects = False
+
+# List of effect I've chosen to make an example for
+effects = [
+    'fan_mode',
+    'fan_speed'
+]
+
+mode_manual = 0x01
+mode_automatic = 0x01
+
+game_mode_balanced = 0x00
```

Comment:
See earlier comment. It seems each interface to the API needs to "know" these exact bytes rather then passing an integer/object. May we "hardcode" them into  the Python library and/or daemon so they can be validated?

For example, the reactive has a `time` and needs to be one of `REACTIVE_` values.